### DISCUSSION TITLE
Series Series

### DISCUSSION THREAD

**Discussion by Toblun** [advplyr/audiobookshelf#900] [Ideas]
Sometimes book series get the label "Series Series" 
I like to name my Seriers folders "Example Name Seriers" and when that gets added it add another Seriers.  Maybe remove any Seriers from the name when adding new Seriers. 
Also sometimes the scrapper have Seriers in the name when you scrap then you also get Seriers Seriers. Might be good with a check that removes that also?

**Comment by advplyr** [comment]
I'm not sure what you mean. "Series" is not added to the end of a series when it is parsed.

**Comment by Toblun** [comment]
![2022-08-11 (2)](https://user-images.githubusercontent.com/51099399/184045957-d3826366-96c1-4dcb-b58c-ca06bfaef9a2.png)

**Reply by Toblun** [reply]
Its not added but its in the interface so any serie already named "Name Series" get the " Series Series effect, its usualy fixed by matching them but sometime they get the "Name Series" from matching then if you want it fixed you have to manual edit every book.
